Packages.props

```diff
@@ -5,7 +5,7 @@
 
         MUST SPECIFY IN CSPROJ AS <PackageReference Name="<depName>" />
     -->
-    <PackageReference Update="CluedIn.Connector.Common" Version="3.4.0-*" />
+    <PackageReference Update="CluedIn.Connector.Common" Version="3.5.0-alpha.3" />
```

Comment:
IDK. This was changed to pull support for underscore without publishing new release version of `CluedIn.Connector.Common` package. If it could be handled other ways, lets change.